service/src/main/kotlin/io/provenance/api/frameworks/config/RateLimiterConfig.kt

```diff
@@ -0,0 +1,11 @@
+package io.provenance.api.frameworks.config
+
+import com.google.common.util.concurrent.RateLimiter
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class RateLimiterConfig {
+    @Bean
+    fun rateLimiter() = RateLimiter.create(5.0)
```

Comment:
Should we make this value a Spring property?

Reply:
Personally I prefer not to unless test / prod vary largely and in this case prod is the upper limit (an i suspect test would have much fewer calls). happy to add it if you'd really like it, though

Reply:
I don't feel that strongly about it so let's leave it be

Reply:
:nice: